dockers/nvidia/Dockerfile

```diff
@@ -12,52 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM nvcr.io/nvidia/cuda:11.1.1-runtime-ubuntu20.04
+# https://docs.nvidia.com/deeplearning/frameworks/pytorch-release-notes/rel_21-03.html#rel_21-03
+FROM nvcr.io/nvidia/pytorch:20.10-py3
```

Comment:
isn't it better to use LTS version? :)

Reply:
NGC does not have LTS tag, this shall be Ubuntu 18.04